include/ddwaf.h

```diff
@@ -414,6 +414,27 @@ bool ddwaf_builder_remove_config(ddwaf_builder builder, const char *path, uint32
  **/
 ddwaf_handle ddwaf_builder_build_instance(ddwaf_builder builder);
 
+/**
+ * ddwaf_builder_get_config_paths
+ *
+ * Provides an array of the currently loaded paths, optionally matching the
+ * regex provided in filter. In addition, the count is provided as the return
+ * value, allowing paths to be nullptr.
+ *
+ * @param builder Builder to perform the operation on. (nonnull)
+ * @param paths The object in which paths will be returned, as an array of
+ *        strings. If NULL, only the count is provided. (nullable)
+ * @param filter An optional string type regex to filter the provided paths. (nullable).
```

Comment:
the documentation should specify the regex is unanchored (it's tested for all positions)